fetch/nosniff/resources/nosniff-line-tabulation.asis

```diff
@@ -0,0 +1,6 @@
+HTTP/1.1 200 YOU HAVE NO POWER HERE
+Content-Length: 22
+Content-Type: x/x
+X-Content-Type-options: nosniff,
```

Comment:
Maybe call this vertical-tab instead of line-tabulation?  At least I've always seen it caller vertical tab when looking at ascii tables and the like.